### PR TITLE
improvement: allow to ping or jump to LMS map from customernodesbox

### PR DESCRIPTION
**Pull request by interduo**

### CODE REVIEW

File: templates/default/customer/customernodesbox.html [hunk 1]
```diff
@@ -73,29 +73,28 @@
...
 										class="{if ($assignment.dateto < $smarty.now && $assignment.dateto) || ($assignment.liabilityid && $assignment.at < $smarty.now && $assignment.at > 365)} blend{elseif ($assignment.datefrom > $smarty.now && $assignment.datefrom && $assignment.period) && !($assignment.suspended)} alertblend{elseif ($assignment.suspended)}suspended{/if}
 										{if !$assignment.commited} lms-ui-assignment-not-commited{/if}">{$assignment.name}</span></a>{if !$assignment@last}, {/if}{/foreach})
 							{/if}
 							{if $node.location}<BR>
-								<span class="nobr">
+								<span class="nobr" class="lms-ui-ignore-target-url">
 									{if $node.teryt}
-										{t a=$node.location|truncate:60:"&hellip;":true}$a (TERRIT){/t}
+										{button icon="map" type="link" href="?m=netdevmap&nodeid={$node.id}"} {t a=$node.location|truncate:60:"&hellip;":true}$a (TERRIT){/t}</a>
 									{else}
```
Comment (chilek) [inline templates/default/customer/customernodesbox.html:96]: Dlaczego nie `...|trunescape:60`?

File: templates/default/customer/customernodesbox.html [hunk 2]
```diff
@@ -73,29 +73,28 @@
 			<div class="lms-ui-tab-table-row{if !$node.access} blend{/if}" data-target-url="?m=nodeinfo&id={$node.id}">
 				<div class="lms-ui-tab-table-wrapper col-7">
 					<div class="lms-ui-tab-table-wrapper col-3">
-						<div class="lms-ui-tab-table-column name">
+						<div class="lms-ui-tab-table-column name lms-ui-ignore-target-url">
 							{if $node.lastonline}
 								{if ($smarty.now-$node.lastonline) > $lastonline_limit}
-									<i class="lms-ui-icon-nodeoff" {tip a=$node.lastonlinedate text="Last online: \$a"}></i>
+									{button icon="nodeoff" type="link" onclick="ping_popup('{$node.ip}')" label=$node.name}
```
Comment (chilek) [inline templates/default/customer/customernodesbox.html:79]: Dlaczego ty znika `last online ...`?

Comment (interduo) [reply templates/default/customer/customernodesbox.html]: https://github.com/chilek/lms/pull/2227/commits/9fac526c78e07bfaaf28459c3c0fb61ba56edff4

**Comment by chilek** [comment]
10q

**Pull request merged**